dts/common/nordic/nrf54h20_enga.dtsi

```diff
@@ -243,28 +396,344 @@
 				port = <9>;
 			};
 
+			comp: comparator@983000 {
+				compatible = "nordic,nrf-comp";
+				reg = <0x983000 0x1000>;
+				status = "disabled";
+				interrupts = <387 NRF_DEFAULT_IRQ_PRIORITY>;
+				#io-channel-cells = <1>;
+			};
+
+			temp: temperature-sensor@984000 {
+				compatible = "nordic,nrf-temp";
+				reg = <0x984000 0x1000>;
+				interrupts = <388 NRF_DEFAULT_IRQ_PRIORITY>;
+				status = "disabled";
+			};
+
+			nfct: nfct@985000 {
+				compatible = "nordic,nrf-nfct";
+				reg = <0x985000 0x1000>;
+				status = "disabled";
+				interrupts = <389 NRF_DEFAULT_IRQ_PRIORITY>;
+			};
+
+			qdec130: qdec@994000 {
+				compatible = "nordic,nrf-qdec";
+				reg = <0x994000 0x1000>;
+				status = "disabled";
+				interrupts = <404 NRF_DEFAULT_IRQ_PRIORITY>;
+			};
+
+			qdec131: qdec@995000 {
+				compatible = "nordic,nrf-qdec";
+				reg = <0x995000 0x1000>;
+				status = "disabled";
+				interrupts = <405 NRF_DEFAULT_IRQ_PRIORITY>;
+			};
+
 			grtc: grtc@99c000 {
 				compatible = "nordic,nrf-grtc";
 				reg = <0x99c000 0x1000>;
 				status = "disabled";
 				cc-num = <16>;
 			};
 
+			timer130: timer@9a2000 {
+				compatible = "nordic,nrf-timer";
+				reg = <0x9a2000 0x1000>;
+				status = "disabled";
+				cc-num = <6>;
+				interrupts = <418 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-bit-width = <32>;
+				prescaler = <0>;
+			};
+
+			timer131: timer@9a3000 {
+				compatible = "nordic,nrf-timer";
+				reg = <0x9a3000 0x1000>;
+				status = "disabled";
+				cc-num = <6>;
+				interrupts = <419 NRF_DEFAULT_IRQ_PRIORITY>;
+				max-bit-width = <32>;
+				prescaler = <0>;
+			};
+
+			i2c130: i2c@9a5000 {
+				compatible = "nordic,nrf-twim";
+				reg = <0x9a5000 0x1000>;
+				status = "disabled";
```

Comment:
All I2C nodes are lacking `easydma-maxcnt-bits = <15>;`.

Reply:
would you mind sending a patch?

Reply:
wait, I see this hasn't been merged, I'll do it myself